src/views/public/PartyDetails.vue

```diff
@@ -111,7 +133,7 @@ export default {
   }
 
   .downvote {
-    top: 12px;
+    top: 8px;
```

Comment:
Did you check that this will not affect the page on desktop view?

Reply:
@adetola-ralph  Yeah it wouldn't. The @screen covers lg upwards.